doc/ProvisionReleaseBuild.md

```diff
@@ -76,65 +95,90 @@ sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/selinux/config
 
    [enclosure-1]
    ```
-### 6. Bootstrap Node
+### 4. Bootstrap Node
    ```
     provisioner setup_provisioner srvnode-1:$(hostname -f) \
     --logfile --logfile-filename /var/log/seagate/provisioner/setup.log --source rpm \
     --config-path ~/config.ini --dist-type bundle --target-build ${CORTX_RELEASE_REPO}
    ```
-### 7. Prepare Pillar Data
-```
-provisioner configure_setup ./config.ini 1
-salt-call state.apply components.system.config.pillar_encrypt
-provisioner confstore_export
-```
+### 5. Prepare Pillar Data
+   ```
+    provisioner configure_setup ./config.ini 1
+    salt-call state.apply components.system.config.pillar_encrypt
+    provisioner confstore_export
+   ```
 
 ## Non-Cortx Group: System & 3rd-Party Softwares
-### 1. Non-Cortx Group: System & 3rd-Party Softwares
 
-```provisioner deploy_vm --setup-type single --states system```
+- ### Non-Cortx Group: System & 3rd-Party Softwares
 
-### 2. Prereq component group
+    ```
+    provisioner deploy_vm --setup-type single --states system
+    ```
 
-``` provisioner deploy_vm --setup-type single --states prereq ```
+- ### Prereq component group
+
+    ```
+    provisioner deploy_vm --setup-type single --states prereq
+    ```
 
 ## Cortx Group: Utils, IO Path & Control Path
 
-### 1. Utils component
+- ### Utils component
 
-``` provisioner deploy_vm --setup-type single --states utils ```
+    ```
+    provisioner deploy_vm --setup-type single --states utils
+    ```
 
-### 2. IO path component group
+- ### IO path component group
 
-``` provisioner deploy_vm --setup-type single --states iopath ```
+    ```
+    provisioner deploy_vm --setup-type single --states iopath
+    ```
 
-### 3. Control path component group
+- ### Control path component group
 
-``` provisioner deploy_vm --setup-type single --states controlpath ```
+    ```
+    provisioner deploy_vm --setup-type single --states controlpath
+    ```
 
 ## Cortx Group: HA
 
-### 1. HA component group
+- ### HA component group
 
-``` provisioner deploy_vm --setup-type single --states ha ```
+    ```
+    provisioner deploy_vm --setup-type single --states ha
+    ```
 
 ## Start cluster (irrespective of number of nodes):
 
-### 1. Execute the following command on primary node to start the cluster:
+- ### Execute the following command on primary node to start the cluster:
 
-``` cortx cluster start ```
+    ```
+    cortx cluster start
+    ```
 
-### 2. Verify Cortx cluster status:
+- ### Verify Cortx cluster status:
 
-``` hctl status ```
+    ```
+    hctl status
+    ```
 
 ## Disable the Firewall
 
 ```
-systemctl stop firewalld
-systemctl disable firewalld
+    systemctl stop firewalld
+    systemctl disable firewalld
 ```
 
+## Cleanup temporary repos
```

Comment:
This should be after pre-requisite package installation.

Reply:
done

Reply:
Cleaning up repo should be after -  Install Provisioner API and requisite packages steps. Refer - https://github.com/Seagate/cortx-prvsnr/wiki/Deploy-VM-Hosted-Repo